src/plone/restapi/tests/test_tus.py

```diff
@@ -105,6 +105,7 @@ def test_tus_post_initialization(self):
         stored_metadata = upload.metadata()
         self.assertEqual(stored_metadata,
                          {u'length': 8, u'mode': u'create'})
+        upload.cleanup()
```

Comment:
@sunew can't we just add this to the test tear down method?

Reply:
the `upload` or the `tus` variables are local for each test case. The TUDUpload instance is created in different ways in different testcases. 
One option woul be to have a tus=None in the setUp() and then do
`self.tus = TUSUpload('myuid', {'length': 2048})`
instead of 
`tus = TUSUpload('myuid', {'length': 2048})`
(and rename ´upload´ to ´tus´ in some cases)

But i don't know if that's a nice thing to do...?

Reply:
@sunew I see. I guess then your version is cleaner and easier to understand. Thanks for the clarification.